src/audio/Kconfig

```diff
@@ -725,4 +725,10 @@ config COMP_MODULE_SHARED_LIBRARY_BUILD
 	  Select if you want to build modules as shared objects that can be used to run
 	  pipelines on the host with the testbench or the ALSA plugin.
 
+config DISABLE_DESCRIPTOR_SPLIT
+	bool "Disable descriptor split"
+	default n
```

Comment:
does it depend on `!FORCE_DMA_COPY_WHOLE_BLOCK`?

Reply:
NO, this cahnge required for only AMD platfroms & for AMD platforms it is not enable FORCE_DMA_COPY_WHOLE_BLOCK.

Reply:
@bandamadhu I think that aligns with what @lyakh proposes. Adding a "depends on !FORCE_DMA_COPY_WHOLE_BLOCK" ensure this option is not enabled in configurations where it does not work. On AMD platforms this check will be true and DISABLE_DESCRIPTOR_SPLIT can be selected.